src/amo/reducers/categories.js

```diff
@@ -158,3 +159,55 @@ export default function reducer(
       return state;
   }
 }
+
+export const getCategories = (
+  categoriesState: CategoryMapType,
+  appName: string,
+  addonType: string,
+): Array<ExternalCategory> | null => {
+  invariant(categoriesState, 'categories state does not contains the app name');
+  invariant(appName, 'app name cannot be empty');
+  invariant(addonType, 'addon type cannot be empty');
+
+  if (categoriesState[appName] && categoriesState[appName][addonType]) {
+    return Object.keys(categoriesState[appName][addonType]).map(
+      (key) => categoriesState[appName][addonType][key],
```

Comment:
I might be missing something here, but can't you just directly return `Object.values(categoriesState[appName][addonType])`?  (or cast it into an list)

Reply:
The conversation with the original contributor ended noting that there would be a flow error - [see comment](https://github.com/mozilla/addons-frontend/pull/9253/files/078edad28569278ad7727b2638c095969add3863#r410454461)

However, I was wondering whether the following updates would be a fix (see latest commit): 
(1) Remove `getCategories`
(2) Revert changes made in `Categories` component
(3) Update `getCategoryNames` by replacing `let categories = getCategories(.....)` with `const categories = Object.values(categoriesState[appName][addonType]);`

cc: @bobsilverberg